edit/style.go

```diff
@@ -7,19 +7,19 @@ var (
 	//styleForPrompt           = ""
 	//styleForRPrompt          = "inverse"
 	styleForCompleted        = styles{"dim"}
+	styleForCompletedHistory = styles{"dim"}
 	styleForMode             = styles{"bold", "lightgray", "magenta"}
 	styleForTip              = styles{}
-	styleForCompletedHistory = styles{"dim"}
 	styleForFilter           = styles{"underlined"}
 	styleForSelected         = styles{"inverse"}
 	styleForScrollBarArea    = styles{"magenta"}
 	styleForScrollBarThumb   = styles{"magenta", "inverse"}
 	styleForSideArrow        = styles{"inverse"}
 
-	// Use black text on white for completion listing.
-	styleForCompletion = styles{"black", "bg_white"}
-	// Use white text on black for selected completion.
-	styleForSelectedCompletion = "inverse"
+	// Use white text on black background for completion listing.
+	styleForCompletion = styles{"white", "bg-black"}
+	// Use black text on white background for selected completion.
+	styleForSelectedCompletion = styles{"black", "bg-white"}
 )
```

Comment:
Well, my intention was to use no style for `styleForCompletion`, and reverse video for selected completion, to agree with other listings. Sorry if this wasn't clear. :(